cob_bringup/bundles/schunk_powercube_legacy.launch

```diff
@@ -1,23 +1,27 @@
 <?xml version="1.0"?>
```

Comment:
do we still have supported robots using `legacy` version of schunk components? if not, we can get rid of this.